src/components/Dashboard/panels/ANJForm.js

```diff
@@ -25,10 +25,24 @@ const ANJForm = React.memo(function ANJForm({
     valueBN: bigNum(0),
     error: null,
   })
-  const { anjToken } = useCourtConfig()
   const theme = useTheme()
+  const { anjToken } = useCourtConfig()
   const inputRef = useSidePanelFocusOnReady()
 
+  const handleEditMode = useCallback(
+    editMode => {
+      setAmount(amount => ({
+        ...amount,
+        value: formatUnits(amount.valueBN, {
```

Comment:
@fabriziovigevani Something that is not nice about this is that decimals get truncated after the blur.

For example, if I wanted to _exactly_ withdraw `1.23456789`, `formatUnits()` changes this to `1.23` when I blur the input.

Reply:
Good catch! Fixing it in https://github.com/aragon/court-dashboard/pull/131